src/Quotations.ts

```diff
@@ -105,13 +105,48 @@ export function extractFromHtml(messageBody: string): ExtractFromHtmlResult {
   if (numberOfCheckpoints >= NodeLimit)
     return { body: messageBody, didFindQuote: false, isTooLong: true };
 
-  const quotationCheckpoints = new Array<boolean>(numberOfCheckpoints);
-  const messagePlainText = preprocess(elementToText(xmlDocument), "\n", ContentTypeTextPlain);
+  let extractQuoteHtml = extractQuoteHtmlViaMarkers(numberOfCheckpoints, xmlDocument, false);
+  if (extractQuoteHtml.error)
+    return { body: messageBody, didFindQuote: false, isTooLong: true };
+
+  // Make sure we did not miss a quote due to some parsing error
```

Comment:
Is the parsing error here related to block tags?
What happens exactly?

Reply:
Sometime we receive messages where the quotation marker is before a div.
If we use the div as a \n we end up with the wrong markers. 
The quotation line is consider as one line of quotation and one line of "possible conversation message".

The goal is to first try it with the current split and if no quote was found try again while considering the block tag as inline